Makefile

```diff
@@ -81,6 +81,10 @@ local-kc-run: local-kc-build
 	@echo "Starting test local app container"
 	@docker-compose -f docker-compose.test.yml up -d
 
+local-kc-run-only:
+	@echo "Starting test local app container"
+	@docker-compose -f docker-compose.test.yml up -d 
```

Comment:
This looks to me like the same as local-kc-run? What is it for?

Reply:
Without the build

Reply:
hated this name but I didn't want to change existing commands I might have preferred

build
run
build-and-run